rasa/utils/tensorflow/models.py

```diff
@@ -71,11 +71,29 @@ class RasaModel(TmpKerasModel):
     Cannot be used as tf.keras.Model.
     """
 
+    def get_config(self):
+        """Returns the config of the layer.
+
+        A layer config is a Python dictionary (serializable)
+        containing the configuration of a layer.
+        The same layer can be reinstantiated later
+        (without its trained weights) from this configuration.
+
+        The config of a layer does not include connectivity
+        information, nor the layer class name. These are handled
+        by `Network` (one layer of abstraction above).
+
+        Returns:
+            Python dictionary.
+        """
+        # TODO: figure out whether it can/needs to be implemented
```

Comment:
This can be removed I suppose?

Reply:
no, this is smth I don't understand still